components/automate-ui/src/app/page-components/search-bar/search-bar.component.html

```diff
@@ -41,7 +41,13 @@
         (keyup)="handleInput($event.key, hint)"
         class="list-item category
               {{ highlightedIndex === i ? 'selected' : '' }}">
-        {{ hint }}
+        <div *ngIf="hint.icon">
+          <div class="suggestion-name-with-icon">{{hint.title}}</div>
+          <chef-icon> {{hint.icon}} </chef-icon>
+        </div>
```

Comment:
If the suggestion has an icon associated to it display it to the left of the label.